[2025-03-20T14:40:01.494+0000] {processor.py:186} INFO - Started process (PID=7887) to work on /opt/airflow/dags/dag.py
[2025-03-20T14:40:01.496+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-03-20T14:40:01.498+0000] {logging_mixin.py:190} INFO - [2025-03-20T14:40:01.498+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-03-20T14:40:01.537+0000] {processor.py:925} INFO - DAG(s) 'mi_nuevo_dag' retrieved from /opt/airflow/dags/dag.py
[2025-03-20T14:40:01.694+0000] {logging_mixin.py:190} INFO - [2025-03-20T14:40:01.693+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-20T14:40:01.889+0000] {logging_mixin.py:190} INFO - [2025-03-20T14:40:01.888+0000] {dag.py:4180} INFO - Setting next_dagrun for mi_nuevo_dag to 2025-03-19 00:00:00+00:00, run_after=2025-03-20 00:00:00+00:00
[2025-03-20T14:40:01.920+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.436 seconds
[2025-03-20T14:40:32.119+0000] {processor.py:186} INFO - Started process (PID=7945) to work on /opt/airflow/dags/dag.py
[2025-03-20T14:40:32.121+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-03-20T14:40:32.123+0000] {logging_mixin.py:190} INFO - [2025-03-20T14:40:32.122+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-03-20T14:40:32.151+0000] {processor.py:925} INFO - DAG(s) 'mi_nuevo_dag' retrieved from /opt/airflow/dags/dag.py
[2025-03-20T14:40:32.174+0000] {logging_mixin.py:190} INFO - [2025-03-20T14:40:32.174+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-20T14:40:32.207+0000] {logging_mixin.py:190} INFO - [2025-03-20T14:40:32.207+0000] {dag.py:4180} INFO - Setting next_dagrun for mi_nuevo_dag to 2025-03-19 00:00:00+00:00, run_after=2025-03-20 00:00:00+00:00
[2025-03-20T14:40:32.419+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.311 seconds
[2025-03-20T14:41:02.821+0000] {processor.py:186} INFO - Started process (PID=8003) to work on /opt/airflow/dags/dag.py
[2025-03-20T14:41:02.825+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-03-20T14:41:02.831+0000] {logging_mixin.py:190} INFO - [2025-03-20T14:41:02.829+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-03-20T14:41:02.900+0000] {processor.py:925} INFO - DAG(s) 'mi_nuevo_dag' retrieved from /opt/airflow/dags/dag.py
[2025-03-20T14:41:02.934+0000] {logging_mixin.py:190} INFO - [2025-03-20T14:41:02.933+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-20T14:41:02.986+0000] {logging_mixin.py:190} INFO - [2025-03-20T14:41:02.985+0000] {dag.py:4180} INFO - Setting next_dagrun for mi_nuevo_dag to 2025-03-19 00:00:00+00:00, run_after=2025-03-20 00:00:00+00:00
[2025-03-20T14:41:03.252+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.453 seconds
[2025-03-20T14:41:33.720+0000] {processor.py:186} INFO - Started process (PID=8061) to work on /opt/airflow/dags/dag.py
[2025-03-20T14:41:33.724+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-03-20T14:41:33.730+0000] {logging_mixin.py:190} INFO - [2025-03-20T14:41:33.729+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-03-20T14:41:33.825+0000] {logging_mixin.py:190} INFO - [2025-03-20T14:41:33.812+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 2, in <module>
    from dags.dag import DAG
ModuleNotFoundError: No module named 'dags'
[2025-03-20T14:41:33.830+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-03-20T14:41:33.887+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.194 seconds
[2025-03-20T14:42:04.449+0000] {processor.py:186} INFO - Started process (PID=8119) to work on /opt/airflow/dags/dag.py
[2025-03-20T14:42:04.452+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-03-20T14:42:04.458+0000] {logging_mixin.py:190} INFO - [2025-03-20T14:42:04.457+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-03-20T14:42:04.546+0000] {logging_mixin.py:190} INFO - [2025-03-20T14:42:04.527+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 2, in <module>
    from dags.dag import DAG
ModuleNotFoundError: No module named 'dags'
[2025-03-20T14:42:04.552+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-03-20T14:42:04.647+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.222 seconds
[2025-03-20T14:42:34.932+0000] {processor.py:186} INFO - Started process (PID=8177) to work on /opt/airflow/dags/dag.py
[2025-03-20T14:42:34.934+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-03-20T14:42:34.936+0000] {logging_mixin.py:190} INFO - [2025-03-20T14:42:34.935+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-03-20T14:42:34.957+0000] {logging_mixin.py:190} INFO - [2025-03-20T14:42:34.952+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 2, in <module>
    from dags.dag import DAG
ModuleNotFoundError: No module named 'dags'
[2025-03-20T14:42:34.960+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-03-20T14:42:34.989+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.067 seconds
